network/load-balancer/reference-content/configuring-frontends.mdx

```diff
@@ -0,0 +1,90 @@
+content:
+  h1: Configuring your Load Balancer's frontend
+  paragraph: Learn how to choose different settings and configurations for your Load Balancer's frontend depending on your needs
+tags: load-balancer
+categories: 
+  - load-balancer
+  - network
+dates:
+  validation: 2023-06-05
+  posted: 2023-06-05
+---
+
+## Frontend overview
+
+Every Load Balancer must have at least one frontend, itself attached to a backend, in order to be functional.
+
+You can [create a frontend at the same time as creating the Load Balancer itself](/network/load-balancer/how-to/create-frontends-backends/#how-to-create-a-frontend-during-creation-of-your-load-balancer), or create an "empty" Load Balancer and [create/add a frontend(s) later](/network/load-balancer/how-to/create-frontends-backends/#how-to-create-a-frontend-after-creation-of-your-load-balancer). 
+
+When creating a frontend and backend at the same time as creating the Load Balancer, you have access only to minimum configuration settings for the frontend:
+- Frontend name
+- Frontend port
+
+When creating a new frontend (or editing an existing one) **after** creating the Load Balancer, you have access to additional settings:
+- Frontend name
+- Frontend port
+- SSL certificate
+- Backend to attach
+- Enable/disable HTTP/3
+
+You can [edit any of these settings](/network/load-balancer/how-to/manage-frontends-and-backends/#how-to-edit-frontends) after creating the frontend, whenever you wish.
+
+You can create multiple frontends to listen on different ports, and connect them to different backends as required. You can also set up custom [ACL rules](/network/load-balancer/how-to/create-manage-acls/) and [routes](/network/load-balancer/how-to/create-manage-routes/) for each frontend.
+
+## Frontend name
+
+You can use an automatically-generated random name suggested by the console, or choose your own name for the frontend.
+
+## Port
+
+The frontend listens on the port you specify, and forwards received requests to the backend it is attached to.
+
+For HTTP, a frontend will typically listen on port 80, while for HTTPS it will typically listen on port 443.
+
+The port number must be between 1 and 65535.
+
+## SSL certificate
+
+<Message type="note">
+
+You can only add an SSL certificate to a frontend after creation of the Load Balancer itself.
+
+</Message>
+
+You can add an SSL/TLS certificate to your frontend to allow clients to securely connect to the Load Balancer via an encrypted connection. This is necessary for [SSL bridging and SSL offloading configurations](/network/load-balancer/reference-content/ssl-bridging-offloading-passthrough/).
+
+When creating a certificate you can generate a Let's Encrypt certificate directly from the console, or import an existing certificate. This could be from a third party Certificate Authority (CA) other than Let's Encrypt, or be a self-signed certificate.
```

Comment:
```suggestion
When creating a certificate you can generate a Let's Encrypt certificate directly from the console, or import an existing certificate. This could be from a third-party Certificate Authority (CA) other than Let's Encrypt, or be a self-signed certificate.
```